cli/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ _Released 07/18/2023 (PENDING)_
 
 **Bugfixes:**
 
+- Fixed invalid stored preference for in-app notifications.  Addresses [#27228](https://github.com/cypress-io/cypress/issues/27228).
```

Comment:
Should this be "Fixes ..."?

Reply:
Also, might want to phrase in a way that would be obvious to the user the impact they would see from adopting this version: "Fixes a potential application crash when enabling in-app notifications."

Reply:
See: https://github.com/cypress-io/cypress/pull/27237/commits/986398ae136238cd6907ca9bfa34a04066616b33